mitreattack/navlayers/generators/usage_generator.py

```diff
@@ -123,6 +123,8 @@ def generate_layer(self, match):
         if self.domain == 'enterprise':
             output_layer.description = f"Enterprise techniques used by {matched_obj.name}, " \
                                        f"ATT&CK {matched_obj.type} {a_id}"
-        else:
+        elif self.domain == 'mobile':
             f"Mobile techniques used by {matched_obj.name}, ATT&CK {matched_obj.type} {a_id}"
+        else:
+            f"ICS techniques used by {matched_obj.name}, ATT&CK {matched_obj.type} {a_id}"
```

Comment:
- Couldn't this entire description block just be done in the format string? 
  ```python
   f"{self.domain.capitalize()} techniques used by [...]"
   ```
   Note: ICS domain may need special handling despite this since it is entirely uppercase, so this solution may not actually be better depending on how the domain strings are stored.
- I believe this wouldn't work for Mobile and ICS even in the current implementation since the format string isn't actually assigned. Unless I'm missing some sort of ternary operator. Was this actually tested for those domains?

Reply:
It was tested... I just only tested for the script not crashing, and didn't directly check the output after verifying it for enterprise (which was definitely an oversight). I've fixed that and simplified it some.